src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const fare = 40;
+
+  if (days >= 7) {
+    return ((days * fare) - 50);
+  } else {
+    if (days >= 3) {
+      return ((days * fare) - 20);
```

Comment:
Доречніше використати `else if`:
```suggestion
  } else if (days >= 3) {
      return ((days * fare) - 20);
```